src/components/NftRewards/QuantitySelector.tsx

```diff
@@ -0,0 +1,43 @@
+import useMobile from 'hooks/Mobile'
+import { classNames } from 'utils/classNames'
+import { PlusOutlined, MinusOutlined } from '@ant-design/icons'
+import { twJoin } from 'tailwind-merge'
+
+// + / - buttons that appear in top-right of selected NFT cards on project page
+export function QuantitySelector({
+  value,
+  onIncrement,
+  onDecrement,
+}: {
+  value: number
+  onIncrement: VoidFunction
+  onDecrement: VoidFunction
+}) {
+  const isMobile = useMobile()
+  const iconClasses =
+    'h-full w-[27px] flex items-center justify-center hover:bg-haze-600'
+  return (
+    <div
+      className={classNames(
+        'absolute right-1.5 top-1.5 items-center text-sm',
+        'flex justify-between rounded-full bg-haze-400 text-smoke-25',
+        isMobile ? 'h-9 w-3/5' : 'h-7 w-1/2',
+      )}
```

Comment:
**question** why do we need to call `classNames()` here?

Reply:
For the `isMobile` condition